tests/spread/integration/libarchive-tools/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Integration tests for libarchive-tools
+
+execute: |
+  rootfs="$(install-slices libarchive-tools_bins)"
+
+  chroot "${rootfs}/" bsdcat --version
+  chroot "${rootfs}/" bsdcpio --version
+  chroot "${rootfs}/" bsdtar --version
+  chroot "${rootfs}/" bsdunzip --version
+  echo "Hello, World" > "${rootfs}/testfile.txt"
+  chroot "${rootfs}/" bsdtar -cf testfile.tar testfile.txt
+  if [ ! -f "${rootfs}/testfile.tar" ]; then 
+    echo "bsdtar failed to create archive: testfile.tar"
+    exit 1
+  fi 
+  echo "testfile.txt" | chroot "${rootfs}/" bsdcpio -o \
+    > "${rootfs}/testfile.cpio"
+  if [ ! -e "${rootfs}/testfile.cpio" ]; then
+    echo "bsdcpio failed"
+    exit 1
+  fi
+  chroot "${rootfs}/" bsdcat testfile.tar
```

Comment:
I would cat the output of a test file and then ensure that it matches the expected output.